<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VqbInstancesSelect</class>
 <widget class="QWidget" name="VqbInstancesSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Triple</string>
   </property>
   <widget class="QCheckBox" name="checkBoxFilter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="KComboBox" name="cbPredicate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="KComboBox" name="cbType">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="KComboBox" name="cbSubject">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="KComboBox" name="cbRelation">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="KComboBox" name="cbObject">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter a string, integer, date or URI.

Examples:
	foaf:Person
	5
	Knud Moeller
            	&lt;http://purl.org/dc/terms/title&gt;
	</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Subject</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Predicate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Object</string>
    </property>
   </widget>
  </widget>
  <widget class="KEditListBox" name="listBoxConditions">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Graph Pattern</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>244</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Output Variables</string>
   </property>
   <widget class="QListWidget" name="listOutputs">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>44</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonAddToOutput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>83</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add -&gt;</string>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonDown">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move Down</string>
    </property>
    <property name="statusTip">
     <string>Moves variable down one position</string>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move Up</string>
    </property>
    <property name="statusTip">
     <string>Moves variable up one position</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputVars">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output variables:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listVars">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>45</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonRemoveFromOutput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>127</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;- Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Available variables:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbSubject</tabstop>
  <tabstop>cbPredicate</tabstop>
  <tabstop>checkBoxFilter</tabstop>
  <tabstop>cbRelation</tabstop>
  <tabstop>cbObject</tabstop>
  <tabstop>cbType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>updateTriple()</slot>
 </slots>
</ui>
